reactnative/RNPlayground/app/screens/PlaygroundScreen.tsx

```diff
@@ -1,76 +1,108 @@
-import React, { FC } from "react"
+import * as React from "react"
 import { observer } from "mobx-react-lite"
-import { StackScreenProps } from "@react-navigation/stack"
-import { AppStackScreenProps } from "../navigators"
-import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
-import {
-  Text, Screen
-} from "../components"
-import { isRTL } from "../i18n"
+import { BaseScreenProps } from "../navigators"
+import { TextStyle, View, ViewStyle } from "react-native"
+import { Text, Button } from "../components"
 import { colors, spacing } from "../theme"
 import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
+import { useFocusEffect } from "@react-navigation/core"
+import { useStores } from "../models"
 
+interface PlaygroundScreenProps extends BaseScreenProps<"Playground"> {
+}
 
-export const PlaygroundScreen: FC<StackScreenProps<AppStackScreenProps, "Playground">> = observer(function PlaygroundScreen() {
-    const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
+export const PlaygroundScreen: React.FC<PlaygroundScreenProps> = observer((props) => {
+  const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
+  const { playgroundStore } = useStores()
```

Comment:
🎉 Nice work

Leaving some extra info for others: https://mobx.js.org/defining-data-stores.html
_The main responsibility of stores is to move logic and state out of your components into a standalone testable unit that can be used in both frontend and backend JavaScript._